views/facetsWithArticles.hbs

```diff
@@ -103,10 +103,10 @@
 				</ul>
 			</li>
 			<li>
-				<h4>Related content</h4>
+				<h4>Charts</h4>
```

Comment:
Can you use an `h3` here to keep the headings in semantic order? It's preceded by a `h2` currently

Reply:
Corrected